src/Common/Notifications/Template.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Notifications Template
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\Notifications
+ */
+
+namespace TEC\Common\Notifications;
+
+use Tribe__Main;
+use Tribe__Template;
+use TEC\Common\Telemetry\Telemetry;
+
+/**
+ * Class Template
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\Notifications
+ */
+class Template extends Tribe__Template {
+
+	/**
+	 * Stores the instance of the template engine that we will use for rendering the page.
+	 *
+	 * @since TBD
+	 *
+	 * @var Tribe__Template
+	 */
+	protected $template;
+
+	/**
+	 * Get template object.
+	 *
+	 * @since TBD
+	 *
+	 * @return \Tribe__Template
+	 */
+	private function get_template() {
+		if ( empty( $this->template ) ) {
+			$this->template = new Tribe__Template();
+			$this->template->set_template_origin( Tribe__Main::instance() );
+			$this->template->set_template_folder( 'src/admin-views/notifications' );
+			$this->template->set_template_context_extract( true );
+			$this->template->set_template_folder_lookup( false );
+		}
+
+		return $this->template;
+	}
```

Comment:
```suggestion
	/**
	 * Notifications Template Constructor
	 *
	 * @since TBD
	 */
	private function __construct() {
			$this->set_template_origin( Tribe__Main::instance() );
			$this->set_template_folder( 'src/admin-views/notifications' );
			$this->set_template_context_extract( true );
			$this->set_template_folder_lookup( false );
		}
	}
```

Since we are extedning we can use the methods of the parent. Make sure you make this a singleton instance so the container builds it only once.